src/command.c

```diff
@@ -291,7 +291,7 @@ static size_t conf_size(unsigned int topo)
  *
  **************************************/
 static int dv_convert_conv_v0(struct device *dev, struct dmp_cmb *cmb,
-			      struct dmp_dv_kcmdraw __user *user_cmd, size_t size)
+			struct dmp_dv_kcmdraw __user *user_cmd, size_t size)
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket